src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -76,16 +86,12 @@ public static function getExtras()
             'plugins' => self::getPlugins(),
             'active_theme' => self::getActiveTheme(),
             'sync_mode' => StoreKeeperOptions::getSyncMode(),
-            'last_sync_run_date' => self::getLastSyncRunDate(),
-            'last_hook_date' => self::getLastHookDate(),
-            'last_hook_id' => self::getLastWebhookLogId(),
-            'last_task_id' => self::getLastTaskId(),
-            'task_quantity' => TaskModel::countTasks(),
-            'task_failed_quantity' => TaskModel::countFailedTasks(),
-            'task_successful_quantity' => TaskModel::countSuccessfulTasks(),
-            'hook_quantity' => WebhookLogModel::count(),
             'active_capability' => self::getActiveCapabilities(),
             'image_variants' => self::getImageVariants(),
+            'plugin_settings_url' => admin_url('/admin.php?page=storekeeper-settings'),
+            'now_date' => current_datetime()->format(DATE_RFC2822),
```

Comment:
```suggestion
            'now_date' => current_datetime()->format('c'),
```
Maybe that's a better format because it's not language dependent

Reply:
Kept `DATE_RFC2822` as discussed.